driver-core/src/main/com/mongodb/client/model/Aggregates.java

```diff
@@ -181,6 +181,18 @@ public static Bson project(final Bson projection) {
         return new SimplePipelineStage("$project", projection);
     }
 
+    /**
+     * Creates a $set pipeline stage for the specified projection
+     *
+     * @param projection the projection
+     * @return the $set pipeline stage
+     * @see Projections
+     * @mongodb.driver.manual reference/operator/aggregation/set/ $set
+     */
+    public static Bson set(final Bson projection) {
```

Comment:
Given that $set is an alias for $addFields, it seems preferable that this new method would have the same signature as the existing addFields method.

Reply:
Fair enough. am happy to continue with this change if you have the appetite to merge it? Let me know.